apps/admin/src/app/(api)/api/v1/token/route.ts

```diff
@@ -40,13 +40,26 @@ export const POST = async (req: NextRequest) => {
             }
 
             if ((sensorData.needsScript || data.needScript) && sensorData.script) {
+                const additionalData: {
+                    script: string | undefined;
+                    analogRotationPerKwh: number | undefined;
+                } = {
+                    script: undefined,
+                    analogRotationPerKwh: undefined,
+                };
+                if (sensorData.script.split("\n").length === 1 && sensorData.script.match(/^\d+$/)) {
+                    const perRotation = Number.parseInt(sensorData.script);
+                    additionalData.analogRotationPerKwh = perRotation;
+                } else {
+                    additionalData.script = sensorData.script;
+                }
                 waitUntil(log("sensor/sent-script", "info", "sensor-token", "api", { sensorData, data }));
```

Comment:
```suggestion
                waitUntil(log("sensor/sent-script", "info", "sensor-token", "api", { sensorData, data, additionalData }));
```
Könntest das auch additionalAnalogSensorData fürs Log nennen